std/traits.d

```diff
@@ -6515,3 +6515,29 @@ unittest
     @Named("abc") int h;
     static assert(hasUDA!(h, Named));
 }
+
+/**
+Returns: $(D true) iff all types $(D T) are the same.
+*/
+template allSame(T...)
```

Comment:
OK, apparently we don't have any reasonable way to compute this. Should be named `allSameType` so it's not confused with value-trafficking artifacts.